lib/gui/app.js

```diff
@@ -296,3 +297,12 @@ app.controller('HeaderController', function(SelectionStateModel, OSOpenExternalS
   };
 
 });
+
+app.controller('StateController', function($state) {
+
+  /**
+   * @param {string} state - state page
+   */
+  this.is = $state.is;
```

Comment:
Unfortunately they don't work well here because the footer's `display: flex` property is overridden with `display: initial` and changes the footer structure.